source/help/getting-started/managing-members.md

```diff
@@ -45,14 +45,33 @@ The System Admin is typically a member of the IT staff and has all the privilege
 - Ability to change any setting on the Mattermost server available in the System Console
 - Ability to promote and demote other users to and from the System Admin role
 - Ability to make a user account Inactive and to reactivate an Inactive account
- 
 
 #### Inactive 
 
-A System Administrator has the ability to make a user account Inactive by going to **System Console** > **Teams**, and selecting a team to manage. From their they can go to the **User** page for the team, and manage a user's role. 
+A System Administrator has the ability to make a user account Inactive by going to **System Console > Teams**, and selecting a team to manage. From there they can go to the **User** page for the team, and manage a user's role. 
 
 When "Inactive" is selected, the user will no longer be able to log into the system. 
 
 Because Mattermost is designed as a system-of-record, there is not an option to delete users from the Mattermost system, as such an operation could compromise the integrity of message archives. 
 
 An Inactive account can also be reactivated from the System Console.
+
+## Removing a User from a Team or Leaving a Team
+
+#### Remove from Team
+
+Any Team Administrator has the ability to remove a user from a team by going to the three dot **Main Menu > Manage Members**, and selecting **Remove From Team** in the dropdown menu beside a user entry. 
+
+When a user is removed from a team, if they currently have the team open they will be sent back to the team selection page. The team will no longer show up in their teams list. 
+
+Removing a user from the team does not make the account Inactive. The user will still be able to sign in to the site, and join other teams. They will also be able to rejoin the team they were removed from if they receive another invite, or if the team is an ["Open"](http://docs.mattermost.com/help/settings/team-settings.html#allow-anyone-to-join-this-team) team. If the user does rejoin the team, they will no longer belong to the channels they were previously a part of. 
```

Comment:
The `allow-anyone-to-join-this-team` is being renamed, in my PR I propose removing the link